cli/cli.py

```diff
@@ -534,6 +614,29 @@ def get_option_type_info() -> dict[str, Any]:
     return option_type_info
 
 
+def process_tables_options(opts: dict[str, Any]) -> dict[str, Any]:
+    new_tables = []
+    if TABLES_LIST in opts:
+        if not is_omitted(opts[TABLES_LIST]):
+            new_tables.extend(parse_tables_list_file(opts[TABLES_LIST]))
+        opts.pop(TABLES_LIST)
+    if TABLES_DIRECTORY in opts:
+        if not is_omitted(opts[TABLES_DIRECTORY]):
+            new_tables.extend(parse_tables_directory(opts[TABLES_DIRECTORY]))
+        opts.pop(TABLES_DIRECTORY)
```

Comment:
```suggestion
    for option_name, parse_func in ((TABLES_LIST, parse_tables_list_file), (TABLES_DIRECTORY, parse_tables_directory)):
        value = opts.pop(option_name)
        if not is_omitted(value):
            new_tables.extend(parse_func(value))
```
I'm pretty sure those options are going to be in the dict